src/complex/Filter/FilterList.cpp

```diff
@@ -101,8 +101,9 @@ bool FilterList::addPlugin(const std::shared_ptr<IPluginLoader>& loader)
   Uuid pluginUuid = plugin->getId();
   if(m_PluginMap.count(pluginUuid) > 0)
   {
-    throw std::runtime_error(fmt::format("Attempted to add plugin '{}' with uuid '{}', but plugin '{}' already exists with that uuid", plugin->getName(), pluginUuid.str(),
-                                         m_PluginMap[pluginUuid]->getPlugin()->getName()));
+    // throw std::runtime_error(fmt::format("Attempted to add plugin '{}' with uuid '{}', but plugin '{}' already exists with that uuid", plugin->getName(), pluginUuid.str(),
+    //                                      m_PluginMap[pluginUuid]->getPlugin()->getName()));
+    return false;
```

Comment:
We'll still want to report this somehow.

Reply:
I would want to report the error somehow unless the thing that calls this method is doing the reporting. You could report a Result<> type, std::optional (that is C++20)